pcsx2/gui/AppGameDatabase.cpp

```diff
@@ -14,128 +14,19 @@
  */
 
 #include "PrecompiledHeader.h"
+
```

Comment:
Unintended whitespace?

Reply:
I typically group the PCH separate, but I don't really have a strong opinion, looks like it's a 50/50 chance of that style being used just looking at a few .cpp files 🤷